# Type: test
# Subtype: T1
# English Summary: Delete unnecessary asEnum in NN testing
# Chinese Summary: 在 NN 测试中删除不必要的 asEnum
# Entry Number: 24

# === Git Diff Content ===

diff --git a/src/test/java/hex/NeuralNetIrisTest.java b/src/test/java/hex/NeuralNetIrisTest.java
index feae3432b..ae86c44f9 100644
--- a/src/test/java/hex/NeuralNetIrisTest.java
+++ b/src/test/java/hex/NeuralNetIrisTest.java
@@ -57,7 +57,6 @@ public class NeuralNetIrisTest extends TestUtil {
     int epochs = 1000;
     Vec[] data = Utils.remove(_train.vecs(), _train.vecs().length - 1);
     Vec labels = _train.vecs()[_train.vecs().length - 1];
-    labels.asEnum();
     VecsInput input = new VecsInput(data, null);
     Layer[] ls = new Layer[3];
     ls[0] = input;
